docs/modules/ROOT/pages/index.adoc

```diff
@@ -852,179 +850,116 @@ The `validFrom` field MUST exist and it MUST express the date and time when the
 
-[#vc-credentialsubject-identities-uri]
-Identity URI::
-The `identities[?].uri` MUST be present and MUST be a valid URI. It represents an URI that can prove the existence of an account for the user name.
-
-[#vc-credentialsubject-identities-provider]
-Identity provider::
-The `identities[?].provider` MUST be a non empty object. The `identities[?].provider` object contains information about the social provider that is associated with this identity.  A social provider MUST contain at least the following two child properties: `id` and `name`.
-
-[#vc-credentialsubject-identities-provider-id]
-Identity provider ID::
-The `identities[?].provider.id` MUST be present and MUST be a valid URI that, when dereferenced, MUST result in a proof existence of the social provider.
-
-[#vc-credentialsubject-identities-provider-name]
-Identity provider name::
-The `identities[?].provider.name` MUST be present and MUST be a non emtpy string that represents the name of the social provider.
-
-Identity creation date::
-The `identities[?].createdAt` MUST be present and MUST be a valid date-time as specified by https://datatracker.ietf.org/doc/html/rfc3339[RFC 3339]. It represents the date and time when this social account was associated with the _<<_named_actor,named actor>>_. 
-
-====== Identity by crypto wallet
-
-This identity representation presents the details of a crypto wallet account of the _<<_named_actor,named actor>>_. The methods for verifying crypto wallet accounts and their association with the _<<_named_actor,named actor>>_ are outside the scope of this specifiation. An identity by crypto wallet needs to contain at least the following four child properties: `type`, `address`, `provider`, and `createdAt`.
-
-Identity by crypto wallet type:::
-In this case the `identities[?].type` property MUST be present and MUST be `cryptoWallet`.
-
-Identity by crypto wallet address:::
-The `identities[?].address` MUST be present and MUST be a valid crypto wallet address. It represents a unique alphanumeric string that can be used to identity the user's crypto wallet.
-
-[#vc-credentialsubject-identities-cw-provider]
-Identity provider:::
-The `identities[?].provider` MUST be a non empty object. The `identities[?].provider` object contains information about the crypto wallet provider that is associated with this identity.  A crypto wallet MUST contain at least the following two child properties: `id` and `name`.
-
-[#vc-credentialsubject-identities-cw-provider-id]
+[#vc-credentialsubject-verifiedidentity-type]
+Verified identity type::
+The `verifiedIdentities[?].type` property MUST be present and MUST be a non-empty string that defines the type of verification that was performed by the _<<_identity_provider,identity provider>>._ This specification defines the following values which _<<_identity_assertion_consumer,identity assertion consumers>>_ SHOULD be prepared to accept:
```

Comment:
should it be `identity assertion generator` as above

Reply:
@1livv I don't think so, but this does point out that we need to provide a definition of _identity provider,_ as distinct from _identity assertion generator._ Rough definitions of both, which I'll attempt to formalize later today:

* _identity assertion generator:_ akin to the claim generator: what's the software that's producing the identity assertion itself
* _identity provider:_ the actor that is attesting to the identity of the named actor. As stated elsewhere, this might be the identity assertion generator or it might be a third party.

Reply:
@1livv @cyraxx filed issue #161 to call out the need for definitions. Should have a PR later today or tomorrow.